src/pages/homePage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+
+import MovieList from '../constainers/MovieList';
+import Header from '../constainers/Header';
+import Footer from '../constainers/Footer';
+import moviesList from '../utils/data';
+
+function HomePage() {
+  const [movies, setMovies] = useState(moviesList);
+
+  const handleSort = sortBy => {
+    const sortedMovies = [
+      ...movies.sort((a, b) => {
+        const valueA = a[sortBy].toUpperCase();
+        const valueB = b[sortBy].toUpperCase();
+        if (valueA < valueB) {
+          return -1;
+        }
+        if (valueA > valueB) {
+          return 1;
+        }
+        return 0;
+      })
+    ];
```

Comment:
Вычисления лучше выносить в хелперы и потом использовать в компоненах. Так будет сокрытие имплементации, а разработчики будут видеть только то что этот массив отсортирован.

```suggestion
    const sortedMovies = getSortedMovie(movies, sortBy)
```